controllers/comparators/route_comparator.go

```diff
@@ -2,17 +2,32 @@ package comparators
 
 import (
 	v1 "github.com/openshift/api/route/v1"
-	"k8s.io/apimachinery/pkg/api/equality"
+	"reflect"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-type RouteComparator struct {
+func GetMMRouteComparator() ResourceComparator {
+	return func(deployed client.Object, requested client.Object) bool {
+		deployedRoute := deployed.(*v1.Route)
+		requestedRoute := requested.(*v1.Route)
+		return reflect.DeepEqual(deployedRoute.Spec.To, requestedRoute.Spec.To) &&
```

Comment:
isn't better to compare the Route itself than its fields?

Reply:
same as above